src/lib/RegistrationSchema.svelte

```diff
@@ -30,16 +30,16 @@
     }
 
     onMount(async () => {
-        papiPath = (await invoke("get_required_env_variables") as RequiredEnvVariables).papiPath
+        papiPath = (await invoke('get_required_env_variables') as SecretVariables).papiPath
```

Comment:
```suggestion
        papiPath = (await invoke('get_secret_variables') as SecretVariables).papiPath
```
Denne har vell endret navn?